jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/ImageMetadataOutputTest.java

```diff
@@ -43,6 +44,7 @@ public void testFromJson() throws IOException {
     Assert.assertEquals(
         "sha256:3f1be7e19129edb202c071a659a4db35280ab2bb1a16f223bfd5d1948657b6fc",
         output.getImageDigest());
+    Assert.assertTrue(output.isImagePushed());
```

Comment:
Could we also add a test to https://github.com/GoogleContainerTools/jib/blob/6df635ee51bc466c961cdeea23d84e84e87ba4b0/jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/JibBuildRunnerTest.java#L230? To ensure that we are setting the imagePushed metadata variable correctly.